test/extended/cluster/clt.go

```diff
@@ -0,0 +1,126 @@
+package cluster
+
+import (
+	"fmt"
+	"sync"
+
+	g "github.com/onsi/ginkgo"
+
+	reale2e "k8s.io/kubernetes/test/e2e"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var viperConfig string
+
+const (
+	DefaultThreadCount int = 10
+	ChannelSize        int = 1024
+)
+
+var _ = g.Describe("[Feature:Performance][Serial][Fast] Load cluster", func() {
+	defer g.GinkgoRecover()
+	var oc = exutil.NewCLIWithoutNamespace("cl")
```

Comment:
don't be stingy. What is this client for?

Reply:
That client is the one used to process and create templates